kubejs/server_scripts/monicoins/saplings.js

```diff
@@ -59,8 +59,8 @@ ServerEvents.recipes(event => {
     event.shaped(
         Item.of('minecraft:mangrove_propagule', 64), [
         'AA ',
-        ' A ',
-        '   '
+        '   ',
+        ' A '
```

Comment:
??

Reply:
I forgot to sync my repo when making this branch. It was an older commit thats aleardy here